tidb.go

```diff
@@ -257,6 +257,14 @@ func trimSQL(sql string) string {
 		}
 		break
 	}
+	// Trim leading '('. For `(select 1);` is also a query.
```

Comment:
This is only used to check if sql is query in interpreter. And the trimed sql text will not be used outside this function.